lib/soundeck/album_repo.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "soundeck/entities/album"
```

Comment:
With the new container structure, we _don't_ do a brute-force require of every single file in lib/. So now we need a "require what you require" approach when it comes to making these available (of course, this does not apply when injecting dependencies, those will be taken care of through the container auto-registration).